js/apps/admin-ui/src/identity-providers/IdentityProvidersSection.tsx

```diff
@@ -84,26 +84,31 @@ export default function IdentityProvidersSection() {
 
   const [addProviderOpen, setAddProviderOpen] = useState(false);
   const [manageDisplayDialog, setManageDisplayDialog] = useState(false);
-  const [providers, setProviders] =
-    useState<IdentityProviderRepresentation[]>();
+  const [hasProviders, setHasProviders] = useState(false);
   const [selectedProvider, setSelectedProvider] =
     useState<IdentityProviderRepresentation>();
   const { addAlert, addError } = useAlerts();
 
   useFetch(
-    async () => {
-      const provider = await adminClient.realms.findOne({ realm });
-      if (!provider) {
-        throw new Error(t("notFound"));
-      }
-      return provider.identityProviders!;
-    },
+    async () => adminClient.identityProviders.find({ max: 1 }),
```

Comment:
No need to mark this as `async` as the function already returns a `Promise`.
```suggestion
    () => adminClient.identityProviders.find({ max: 1 }),
```